<launch>
    <node pkg="matrice_traj_tracker" type="matrice_traj_tracker_node"
    name="matrice_traj_tracker_node" output="screen">
    <!-- node parameters -->
        <param name="takeoff_height" type="double" value="3.0"/>
        <param name="max_vx" type="double" value="0.2"/>
        <param name="max_vy" type="double" value="0.2"/>
        <param name="max_vz" type="double" value="0.2"/>
        <param name="max_ry" type="double" value="0.2"/>
        <param name="min_vx" type="double" value="0.05"/>
        <param name="min_vy" type="double" value="0.05"/>
        <param name="min_vz" type="double" value="0.05"/>
        <param name="min_ry" type="double" value="0.2"/>
        <param name="watchdog_freq" type="double" value="7.0"/>
        <param name="testing" value="false" />
        <param name="gazebo_sim" value="true" />
        <param name="drone_model" value="m210" />
        <param name="drone_frame" value="firefly/base_link" />
        <param name="speed_reference_mode" value="true" />
        <remap from="/dji_sdk/rc" to="/firefly/joy" />
        <remap from="/dji_sdk/gps_position" to="/firefly/gps" />
    </node>

</launch>

